recipes/hhsuite/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "v3.1.0" %}
+{% set sha256 = "e67527c8dd175f5730ca877aa506bc7aa4178cb0154d5b1a2794e1a484956c83" %}
+
+package:
+  name: hhsuite
+  version: {{ version }}
+
+build:
+  number: 1
+
+source:
+  url: https://github.com/soedinglab/hh-suite/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make
+  host:
+    - openmp
+    - perl
+    - python
+
+  run:
+    - openmp
+
+test:
+  commands:
+    - hhblits -h > /dev/null
+    - hhsearch -h > /dev/null
+    - hhalign -h > /dev/null
+
+about:
+  home: https://github.com/soedinglab/hh-suite
+  license: GPLv3
```

Comment:
please add `license_file: LICENSE`

Reply:
I have troubles to edit this recipe. I do not have the copy of the repository I used to create this fomular anymore. 

If I check out a new bioconda and switch to the hhsuite branch then the recipe folder (recipes/hhsuite) is empty. Also if I try to push some new file then the following error occurs "remote: `Permission to bioconda/bioconda-recipes.git denied to martin-steinegger`.

Also, checking out PR leads to an error 
    
     git fetch origin  pull/13796/head:hhsuite

     remote: Enumerating objects: 8, done.
     remote: Counting objects: 100% (8/8), done.
     remote: Compressing objects: 100% (8/8), done.
     remote: Total 8 (delta 2), reused 0 (delta 0), pack-reused 0
     Unpacking objects: 100% (8/8), done.
     From https://github.com/bioconda/bioconda-recipes
     ! [rejected]            refs/pull/13796/head -> hhsuite  (non-fast-forward)

Do you have any idea how I can edit this recipe again? Thank you for your help!